test/mocha.opts

```diff
@@ -0,0 +1 @@
+--timeout 15000
```

Comment:
why is this here? is this global for all tests? it's better to define it for individual tests or for suites:
see https://mochajs.org/#timeouts

Reply:
Yes, this sets the global timeout. I got impatient after chasing timeout issue after timeout issue.